external/webserver/http_client.c

```diff
@@ -501,6 +534,15 @@ int http_recv_and_handle_request(struct http_client_t *client, struct http_keyva
 			goto errout;
 		}
 	}
+
+	// Check "Connection" header value
+	conn_type = http_keyvalue_list_find(request_params, "Connection");
```

Comment:
http_keyvalue_list_find returns value which matched with key, then I think conn_type is not good name.